Introductory Tests/UDP_basic.ipynb

```diff
@@ -0,0 +1,1402 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #89.    def get_trajectory(mesh, vertices, faces, x):
</pre><p>this function contains a lot of the same code as fitness. Ideally, refactor so that the fitness function calls this?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rasmusmarak/67P-Swarm/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>